### ISSUE TITLE
Hi can't do the pull requests. I have added a simple function for set swype on or off while running

### ISSUE THREAD

**Issue by Dani-07** [SidebarJS/sidebarjs#13]
```
SidebarElement.prototype.setSwype = function (sln_swype) {
			var _this = this;
			if (sln_swype) {
				this.component.addEventListener('touchstart', this.__onTouchStart, { passive: true });
				this.component.addEventListener('touchmove', this.__onTouchMove, { passive: true });
				this.component.addEventListener('touchend', this.__onTouchEnd, { passive: true });
				document.addEventListener('touchstart', this.__onSwipeOpenStart, { passive: true });
				document.addEventListener('touchmove', this.__onSwipeOpenMove, { passive: true });
				document.addEventListener('touchend', this.__onSwipeOpenEnd, { passive: true });

			}else{
				this.component.removeEventListener('touchstart', this.__onTouchStart, {passive: true});
				this.component.removeEventListener('touchmove', this.__onTouchMove, {passive: true});
				this.component.removeEventListener('touchend', this.__onTouchEnd, {passive: true});
				this.container.removeEventListener('transitionend', this.__onTransitionEnd, {passive: true});
				document.removeEventListener('touchstart', this.__onSwipeOpenStart, {passive: true});
				document.removeEventListener('touchmove', this.__onSwipeOpenMove, {passive: true});
				document.removeEventListener('touchend', this.__onSwipeOpenEnd, {passive: true});
			}
        }; 
```

**Comment by lorenzodianni** [comment]
Hi @Dani-07,
can you tell me what is the use case for this feature?

**Comment by Dani-07** [comment]
Hi, yes of course.
I'm doing an app and I have a calendar like this: 
![image](https://user-images.githubusercontent.com/17598391/47573924-c587a100-d93e-11e8-82a9-d4c7c81e8c34.png)

On this screen the swype is used to change the calendar month. For that I remove the swype of sidebar. 
![sin titulo](https://user-images.githubusercontent.com/17598391/47574181-5494b900-d93f-11e8-9d1a-663d5d598b8e.png)

In the menu I activate the sliding of the sidebar again.

**Comment by lorenzodianni** [comment]
Hi @Dani-07, I've just released sidebarjs@5.4.0 with this new feature 🎉 
Thank you!
